account/account.go

```diff
@@ -533,40 +532,37 @@ func (account *Account) WaitForTransactionReceipt(ctx context.Context, transacti
 	}
 }
 
-// AddInvokeTransaction generates an invoke transaction and adds it to the account's provider.
+// SendTransaction can send Invoke, Declare, and Deploy transactions. It provides a unified way to send different transactions.
 //
 // Parameters:
 // - ctx: the context.Context object for the transaction.
-// - invokeTx: the invoke transaction to be added.
+// - txn: this Broadcast Transaction to be sent.
 // Returns:
-// - *rpc.AddInvokeTransactionResponse: The response for the AddInvokeTransactionResponse
+// - interface{} returns required Broadcast txn return value.
```

Comment:
1 - ~~this~~ **the** Broadcast Transaction to be sent.
2 - Wrong type description, it's not "interface{}". Also, let's follow the pattern= returnType: description

Reply:
done